packages/themes/package.json

```diff
@@ -24,14 +24,13 @@
     "build": "yarn clean && bundler bundle src/index.js --name CarbonThemes && node tasks/build.js && bundler check \"scss/*.scss\" && bundler sassdoc \"scss/**/*.scss\""
   },
   "dependencies": {
-    "@carbon/colors": "10.5.0",
-    "@carbon/type": "10.6.0",
-    "@carbon/layout": "10.5.0"
```

Comment:
Did we loose a dependency here?

Reply:
Yeah, I don't know. I'll add it back in!